platformio_esp32_envs.ini

```diff
@@ -22,7 +22,9 @@ build_flags               = ${mqtt_flags.build_flags}
                             -DCONFIG_LWIP_ESP_GRATUITOUS_ARP
                             -DCONFIG_LWIP_GARP_TMR_INTERVAL=30
 monitor_filters           = esp32_exception_decoder
-
+;upload_port               = COM4
+upload_speed              = 921600
```

Comment:
Do not use such fast upload speed as default for all.
Not all USB to serial chips can handle this reliable.
That's the reason those are not included in the PIO definitions and only a single global value with a safe upload speed.

Reply:
yes I know, I made a mistake and will remove it

Reply:
Ah OK, my mistake then...

Reply:
;-) seems that I saw this and already corrected this